packages/material-ui-lab/src/Pagination/Pagination.d.ts

```diff
@@ -1,2 +1,100 @@
-declare const Pagination: any;
-export default Pagination;
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+import { UsePaginationItem } from './usePagination';
+
+export interface PaginationProps
+  extends StandardProps<
+    React.HTMLAttributes<HTMLElement>,
+    PaginationClassKey,
+    'onChange' | 'children'
+  > {
+  /**
+   * Number of always visible pages at the beginning and end.
+   */
+  boundaryCount?: number;
+  /**
+   * The active color.
+   */
+  color?: 'default' | 'primary' | 'secondary';
+  /**
+   * The total number of pages.
+   */
+  count?: number;
+  /**
+   * The page selected by default when the component is uncontrolled.
+   */
+  defaultPage?: number;
+  /**
+   * If `true`, the pagination will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Accepts a function which returns a string value that provides a user-friendly name for the current page.
+   *
+   * For localization purposes, you can use the provided [translations](/guides/localization/).
+   *
+   * @param {string} [type = page] The link or button type to format ('page' | 'first' | 'last' | 'next' | 'previous').
+   * @param {number} page The page number to format.
+   * @param {bool} selected If true, the current page is selected.
+   * @returns {string}
+   */
+  getItemAriaLabel?: (
+    type: UsePaginationItem['type'] | undefined,
+    page: number,
+    selected: boolean,
+  ) => string;
+  /**
+   * If `true`, hide the next-page button.
+   */
+  hideNextButton?: boolean;
+  /**
+   * If `true`, hide the previous-page button.
+   */
+  hidePrevButton?: boolean;
+  /**
+   * Callback fired when the page is changed.
+   *
+   * @param {object} event The event source of the callback.
+   * @param {number} page The page selected.
+   */
+  onChange?: (event: React.ChangeEvent<{}>, page: number) => void;
```

Comment:
Prefer `unknown`. We use it in older types for backwards compat. Newer types should almost always use `unknown`